usaspending_api/awards/v2/views/idvs/activity.py

```diff
@@ -78,11 +80,49 @@
     where   {award_id_column} = {award_id}
 """)
 
+COUNT_ACTIVITY_HIDDEN_SQL = SQL("""
+    with gather_award_ids as (
+        select  award_id,
+                false grandchild
+        from    parent_award
+        where   {award_id_column} = {award_id}
+        union all
+        select  cpa.award_id,
+                true grandchild
+        from    parent_award ppa
+                inner join parent_award cpa on cpa.parent_award_id = ppa.award_id
+        where   ppa.{award_id_column} = {award_id}
+    )
+    select
+        count(ca.id) rollup_contract_count
+    from
+        gather_award_ids gaids
+        inner join awards pa on pa.id = gaids.award_id
+        inner join awards ca on
+            ca.parent_award_piid = pa.piid and
+            ca.fpds_parent_agency_id = pa.fpds_agency_id and
+            ca.type not like 'IDV%'
+            {hide_edges_awarded_amount}
+        left outer join transaction_fpds tf on tf.transaction_id = ca.latest_transaction_id
+        left outer join recipient_lookup rl on rl.duns = tf.awardee_or_recipient_uniqu
+        left outer join agency a on a.id = ca.awarding_agency_id
+        left outer join toptier_agency ta on ta.toptier_agency_id = a.toptier_agency_id
+    {hide_edges_end_date}
+""")
+
 
 def _prepare_tiny_shield_models():
     # This endpoint has a fixed sort.  No need for "sort" or "order".
     models = [copy(p) for p in PAGINATION if p["name"] in ("page", "limit")]
     models.extend([get_internal_or_generated_award_id_model()])
+    x = [
+        {'name': 'hide_edge_cases', 'type': 'boolean', 'optional': True},
+    ]
+    for p in x:
+        p['optional'] = p.get('optional', True)
+        p['key'] = p['name']
+    models.extend(x)
```

Comment:
Perhaps I'm misunderstanding what's going on here, but can the prior 7 lines be boiled down to one?
```
models.append({'key': 'hide_edge_cases', 'name': 'hide_edge_cases', 'type': 'boolean', 'optional': True, 'default': False})
```